src/VisualStudio/LiveShare/Impl/Client/Razor/CSharpLspContainedLanguageProvider.cs

```diff
@@ -37,7 +36,7 @@ internal class CSharpLspContainedLanguageProvider : IContainedLanguageProvider
         }
 
         public IContentType GetContentType(string filePath)
```

Comment:
Once razor switches fully to their LSP based editor in liveshare scenarios, this razor support can be removed.